engine/vm_test.go

```diff
@@ -270,6 +270,15 @@ func TestVM_SetUserOutput(t *testing.T) {
 	})
 }
 
+func TestVM_SetMaxVariables(t *testing.T) {
+	t.Run("ok", func(t *testing.T) {
+		var vm VM
+		vm.SetMaxVariables(10)
+		assert.Equal(t, uint64(10), maxVariables)
+		assert.Equal(t, uint64(10), vm.maxVariables)
+	})
+}
```

Comment:
@coderabbitai Sorry the `SetMaxVariables` first argument is `uint64` so it's impossible to set a negative value !

Reply:
@coderabbitai Ok, If you thinks it's better, the coverage will not be increased but it can be added for understanding. You can add this test case please.